backend/headless/backend.c

```diff
@@ -7,10 +7,17 @@
 #include <wlr/util/log.h>
 #include "backend/headless.h"
 #include "glapi.h"
+#include "util/signal.h"
+
+struct wlr_headless_backend *headless_backend_from_backend(
+		struct wlr_backend *wlr_backend) {
```

Comment:
Should this be `get_headless_backend_from_backend()` to be consistent with the other `get_foo_from_bar()`'s above?

Reply:
The others have a `get_` prefix to prevent clashes with library names (`drm`, `libinput`, `wl`, etc...).